addons/compat_3as/CfgVehicles.hpp

```diff
@@ -0,0 +1,47 @@
+class CfgVehicles {
+    class Helicopter_Base_H;
+    class 3as_laat_Base: Helicopter_Base_H {
+        GVARMAIN(passengerTurrets)[] = {
+            // Left and right ramp seats, side seats, inside standing seats
+            {1}, {2},
+            {3}, {4},
+            {5}, {6},
+            {7}, {8},
+            {9}, {10},
+            {11}, {12},
+            {13}, {15},
+            {16}, {17},
+            {18}, {19},
+            {20}, {21},
+            {22}, {23},
+            {24}, {25},
+            {26}, {27},
+            {28}, {29},
+            {30}, {31},
+            {32}, {33},
+            {34}
+        };
+        GVARMAIN(rampAnims)[] = {{"ramp", 0, 1}};
+
+        EGVAR(helocast,boatPositions)[] = {

```

Comment:
This offset is model specific, since the model center could be the center of the model or it could be slightly below, above, etc.